packages/lesswrong/components/comments/DebateResponseBlock.tsx

```diff
@@ -195,6 +218,16 @@ export const DebateResponseBlock = ({ responses, post, orderedParticipantList, d
             {replyLink}
           </div>
           {replyState}
+          {VoteBottomComponent && <div className={classes.reacts}>
```

Comment:
Can we conditionally display this only if the user is a co-author?  (And maybe also not on their own comments?)

Reply:
Me and Ray disagree. Seems good for all users to see the react activity?